### PR TITLE
WIP: feat: implement drag and drop for the page list container

### PR DESCRIPTION
**Pull request by Palmaswell**

### CODE REVIEW

**Comment by markusoelhafen** [comment]
Hey @Palmaswell ,
thx for the PR, I just looked into it, and I found two issues (from the ux side of things ;) ). 

- [ ] 1. Changing the order of the pages can't be undone. Seems like this action is not pushed to the store smh.
- [ ] 2. The Target Indicator has a weird behaviour. And this weird behaviour is hard to describe, but I'll do my best ;) 
It seems that the dropzone for pages is a different from the dropzone in the element list. I try to make this visible with videos :)
- Dragging the page up doesn't show an indicator like it does in the element list, imediately. It only shows when I drag the page over the page I want it to. While I could live with this implementation, it is not consistent with how dragging is implemented in the element list & for me the element list feels more natural.
drop page up
- Dragging the page down
 seems like the indicator is shown in the wrong place. The indicator is shown above the element, but on release the dragged element is placed below the other element. (I hope the gif illustrates this issue better then my writing ;) )
drag page down

File: src/container/element-list/element-list.tsx [hunk 1]
```diff
@@ -258,15 +254,15 @@ export class ElementList extends React.Component {
...
+		// const node = e.target as Node;
+		// const contains = (target: Node) => (this.ref ? this.ref.contains(target) : false);
 
 		// Only handle key events if either
 		// (1) it is global, thus fires on body
 		// (2) is a node inside the page element list
-		if (e.target !== document.body && !contains(node)) {
-			return;
-		}
+		// if (e.target !== document.body && !contains(node)) {
```
Comment (tilmx) [inline src/container/element-list/element-list.tsx]: Same as above

File: src/container/element-list/element-list.tsx [hunk 2]
```diff
@@ -258,15 +254,15 @@ export class ElementList extends React.Component {
 
 	private handleKeyDown(e: KeyboardEvent): void {
 		const { store } = this.props as { store: Store.ViewStore };
-		const node = e.target as Node;
-		const contains = (target: Node) => (this.ref ? this.ref.contains(target) : false);
+		// const node = e.target as Node;
```
[the remark this answers is not in the document]
Comment (Palmaswell) [reply src/container/element-list/element-list.tsx]: No, removed :)

File: src/components/page-tile/demo.tsx [hunk 3]
```diff
@@ -1,48 +1,55 @@
 import * as React from 'react';
 
-import { Copy } from '../copy';
+// import { Copy } from '../copy';
```
[the remark this answers is not in the document]
Comment (Palmaswell) [reply src/components/page-tile/demo.tsx]: Done 👍

File: src/container/element-list/element-list.tsx (hunk 1 above)
Comment (Palmaswell) [reply src/container/element-list/element-list.tsx]: done 👍

**Comment by Palmaswell** [comment]
Closed by #603

**Pull request closed without merging**